prow/jobs/console/react-components/react-components.yaml

```diff
@@ -0,0 +1,28 @@
+presubmits:
+  kyma-project/console:
+    - name: console-react-components
+      run_if_changed: "^components/react/"
+      branches:
+        - prow-react-components
+      skip_report: true
+      decorate: true
+      path_alias: github.com/akucharska/console
```

Comment:
Wrong alias, it should be `github.com/kyma-project/console`

Alias generally should always point to upstream repository.